components/org.wso2.carbon.siddhi.metrics.core/src/main/java/org/wso2/carbon/siddhi/metrics/core/SiddhiLatencyMetric.java

```diff
@@ -33,22 +33,21 @@
 public class SiddhiLatencyMetric implements LatencyTracker {
     // Using thread local variables to keep the timer track
     // the time of the same execution path by different threads.
-    private static final String METRIC_SUFFIX_LATENCY = "latency";
     private ThreadLocal<Timer> execLatencyTimer;
     private ThreadLocal<Timer.Context> context;
     private String latencyTrackerId;
 
-    public SiddhiLatencyMetric(String name, MetricService metricService, boolean isStatisticEnabled) {
-        this.latencyTrackerId = MetricService.name(name, METRIC_SUFFIX_LATENCY);
+    public SiddhiLatencyMetric(String latencyTrackerId, MetricService metricService, boolean isStatisticEnabled) {
+        this.latencyTrackerId = latencyTrackerId;
         Timer timer = metricService.timer(this.latencyTrackerId, INFO);
-        SiddhiMetricsDataHolder.getInstance().getMetricManagementService().setMetricLevel(latencyTrackerId, OFF);
+        SiddhiMetricsDataHolder.getInstance().getMetricManagementService().setMetricLevel(this.latencyTrackerId, OFF);
         execLatencyTimer = new ThreadLocal<Timer>() {
             protected Timer initialValue() {
             if (isStatisticEnabled) {
-                SiddhiMetricsDataHolder.getInstance().getMetricManagementService().setMetricLevel(latencyTrackerId,
+                SiddhiMetricsDataHolder.getInstance().getMetricManagementService().setMetricLevel(SiddhiLatencyMetric.this.latencyTrackerId,
                         INFO);
             } else {
-                SiddhiMetricsDataHolder.getInstance().getMetricManagementService().setMetricLevel(latencyTrackerId,
+                SiddhiMetricsDataHolder.getInstance().getMetricManagementService().setMetricLevel(SiddhiLatencyMetric.this.latencyTrackerId,
                         OFF);
             }
                 return timer;
```

Comment:
fix indentation

Reply:
Fixed in b4a2bcf444d25031775a679bad76798d70ce7d5b